### PR TITLE
National minimum living wage update

### PR DESCRIPTION
**Pull request by pmanrubia**
paired with @leenagupte 

### Trello card
* [change-first-question-in-national-minimum-wage-smart-answer](https://trello.com/c/11QxrrH1/103-change-first-question-in-national-minimum-wage-smart-answer)

## Description

In the current version it is not possible to calculate the National Minimum Wage between Oct 2015 and Apr 2016. In April 2016, a new "Living Wage" was introduced for people over the age of 25. That means that when you check the minimum/living wage for the current year (Oct 2015 onwards) over 25 year olds are always presented with the new Living Wage.

The solution to this problem is to add a new date range to the past payment option that allows you to select Oct 2015 - Mar 2016.

## Factcheck
[National minium wage fact check](https://smart-answers-pr-2456.herokuapp.com/am-i-getting-minimum-wage/y/past_payment)

## Expected changes

* [URL on GOV.UK](https://www.gov.uk/am-i-getting-minimum-wage/y/past_payment)

### Before
screen shot 2016-04-12 at 17 13 57

### After
screen shot 2016-04-12 at 17 14 04

### CODE REVIEW

File: test/artefacts/am-i-getting-minimum-wage/past_payment/2008-10-01/apprentice_under_19.txt [hunk 1]
```diff
@@ -1,6 +1 @@
-
-
-There was no National Minimum Wage rate for apprentices before 1 October 2010. Your employer doesn't owe you any past payments.
-
-
-
```
Comment (ikennaokpala) [inline test/artefacts/am-i-getting-minimum-wage/past_payment/2008-10-01/apprentice_under_19.txt]: 404 Error appear here..  for 2008-10-01 related pages

I guess this is unintentional and probably generated by mistake.. kindly take note of this.

Comment (leenagupte) [inline test/artefacts/am-i-getting-minimum-wage/past_payment/2008-10-01/apprentice_under_19.txt]: I didn't know to run the script to generate responses and expected results. I shall update the documentation. Thanks.

File: test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb [hunk 2]
```diff
@@ -331,6 +331,41 @@ class AmIGettingMinimumWageTest < ActiveSupport::TestCase
...
+              assert_match(/You are getting the National Living Wage./, outcome_body)
+            end
+          end
+          context 'when he is paid below the minimum/living wage' do
+            setup do
+              add_response 40 # how much it is paid per period
+              add_response 0 # overtime
+              add_response 'no' # accommodation
+            end
+            should ' should reach the below national living wage result outcome ' do
```
Comment (ikennaokpala) [inline test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb]: Is this Intentional ??

`' should reach the below national living wage result outcome '` 

Notice the white spaces

I suppose you want this
`'should reach the below national living wage result outcome'`

Comment (leenagupte) [inline test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb]: Should probably remove the "should" too.

**Comment by ikennaokpala** [comment]
LGTM

**Pull request merged**